run.sh

```diff
@@ -0,0 +1 @@
+/Users/hsum/homebrew/bin/analytics --output /Users/hsum/Desktop/real_time_cfpb/analytics.usa.gov
```

Comment:
For some reason cron likes absolute paths, but I can try it in vagrant and see if that will work.

Sent from my iPhone

> On Nov 16, 2015, at 3:53 PM, Tim Anderegg <notifications@github.com> wrote:
> 
> In run.sh:
> 
> > @@ -0,0 +1 @@
> > +/Users/hsum/homebrew/bin/analytics --output /Users/hsum/Desktop/real_time_cfpb/analytics.usa.gov
> OK, you should be able to just run "analytics --output etc..." without the path, since it is being installed as an NPM package. If you could commit the change to your repo when you get a chance, let me know. Thanks!
> 
> —
> Reply to this email directly or view it on GitHub.
>